src/serviceregistry/manual/examples.md

```diff
@@ -0,0 +1,2356 @@
+  
+  _Query parameter: verbose=true_
+
+  Request body:
+  ~~~
+  {
+    "systemNames": [
+    ],
+    "addresses": [
+    ],
+    "addressType": "",
+    "metadataRequirementList": [
+      {
+        "customizable": true,
+        "type": "temperature",
+        "scales": { "op": "CONTAINS", "value": "Celsius" }
+      }
+    ],
+    "versions": [
+    ],
+    "deviceNames": [
+      "thermometer1", "thermometer2", "thermometer3", "thermometer5"
+    ]
+  }
+  ~~~
+
+  Response body:
+  ~~~
+  {
+    "entries": [
+      {
+        "name": "main-temperature-provider",
+        "metadata": {
+          "type": "temperature",
+          "customizable": true,
+          "scales": [
+            "Fahrenheit",
+            "Kelvin",
+            "Celsius"
+          ]
+        },
+        "version": "2.0.0",
+        "addresses": [
+          {
+            "type": "IPV4",
+            "address": "128.0.1.2"
+          },
+          {
+            "type": "MAC",
+            "address": "12:34:5a:ff:c9:44"
+          }
+        ],
+        "device": {
+          "name": "thermometer5",
+          "metadata": {
+            "scales": [
+              "Fahrenheit",
+              "Kelvin",
+              "Celsius"
+            ],
+            "max-temperature": {
+              "Kelvin": 310,
+              "Celsius": 40,
+              "Fahrenheit": 140
+            },
+            "min-temperature": {
+              "Fahrenheit": 20,
+              "Kelvin": 260,
+              "Celsius": -10
+            },
+            "appliable": [
+              "spring",
+              "summer",
+              "autumn",
+              "winter"
+            ]
+          },
+          "addresses": [
+            {
+              "type": "MAC",
+              "address": "81:ef:1a:44:7b:02"
+            }
+          ],
+          "createdAt": "2024-10-22T10:34:10Z",
+          "updatedAt": "2024-10-22T10:34:10Z"
+        },
+        "createdAt": "2024-10-23T21:43:35Z",
+        "updatedAt": "2024-10-23T21:43:35Z"
+      }
+    ],
+    "count": 1
+  }
+  ~~~
+- **Revoke system:** DELETE 
+/serviceregistry/system-discovery/revoke
+
+Header:
+~~~
+curl -X 'DELETE' \
+  'http://localhost:8443/serviceregistry/system-discovery/revoke' \
+  -H 'accept: */*' \
+  -H 'Authorization: Bearer SYSTEM//temperature-provider1'
+~~~
+
+~~~
+http://localhost:8443/serviceregistry/system-discovery/revoke
+~~~
+
+Reponse code: 200 
+
+## **Service discovery**
+
+- **Register service:** POST /serviceregistry/service-discovery/register
+  
+  Request body:
+  ~~~
+  {
+    "serviceDefinitionName": "celsius-info",
+    "version": "1",
+    "expiresAt": "2030-10-11T14:30:00Z",
+    "metadata": {
+      "frequency": 400
+    },
+    "interfaces": [
+      {
+        "templateName": "generic-https",
+        "protocol": "https",
+        "policy": "NONE",
+        "properties": {
+          "accessAddresses": ["192.168.100.100"],
```

Comment:
should be a typical use case example here. Which means that in case of HTTP or HTTPS this IP should be the same than the provider system's IP. In case of MQTT, it should be different.